packages/ui/src/alert-dialog/alert-dialog.tsx

```diff
@@ -0,0 +1,33 @@
+import type { ReactNode } from "react";
+import * as AlertDialog from "@radix-ui/react-alert-dialog";
+
+// ==============================
+// Radix Components
+// https://github.com/radix-ui/primitives/blob/main/packages/react/alert-dialog/src/index.ts
+// ==============================
+export const Root = AlertDialog.Root;
+export const Trigger = AlertDialog.Trigger;
+export const Portal = AlertDialog.Portal;
+export const Overlay = AlertDialog.Overlay;
+export const Content = AlertDialog.Content;
+export const Action = AlertDialog.Action;
+export const Cancel = AlertDialog.Cancel;
+export const Title = AlertDialog.Title;
+export const Description = AlertDialog.Description;
+
+// ==============================
+// CUSTOM OVERRIDES
+// ==============================
+
+// ==============================
+// CUSTOM COMPONENTS
+// ==============================
+
+export const Body = ({ children }: { children: ReactNode }) => (
```

Comment:
@juliusmarminge I've put the AlertDialog into our ui package because I wanted to add the `Body` component. 

Can be used as regular radix or with `<AlertDialog.Body>` for proper overlay styling & content center.